migtests/tests/import-file/validate

```diff
@@ -18,15 +18,34 @@ EXPECTED = {
 }
 
 T1_TABLE_DATA = set([(1,None,None,"'Hello World'"), (2,None,None,"'Hello World'"), (3,None,None,"'Hello World'")])
+T1_NEWLINE_DATA = set([(1,None,None,"'Hello World'"), (2,None,None,"'Hello\nWorld'"), (3,None,None,"'Hello\nWorld'"), 
```

Comment:
Note that there should not be single quotes around `Hello world` for the `t1_newline.csv`. How come the tests passed 🤔 ?